dojo/problem/helper.py

```diff
@@ -0,0 +1,118 @@
+import json
+import os
+import requests
+
+from django.conf import settings
+
+from dojo.models import Problem, Finding
+
+import logging
+logger = logging.getLogger(__name__)
+
+MEDIA_ROOT = os.getenv('DD_MEDIA_ROOT', '/app/media')
+CACHED_JSON_FILE = os.path.join(MEDIA_ROOT, 'cached_disambiguator.json')
+
+SEVERITY_ORDER = {
+    'Critical': 5,
+    'High': 4,
+    'Medium': 3,
+    'Low': 2,
+    'Info': 1
+}
+
+def validate_json(data):
+    if not isinstance(data, dict):
+        return False
+    for key, value in data.items():
+        if not isinstance(key, str) or not isinstance(value, list):
+            return False
+        if not all(isinstance(item, str) for item in value):
+            return False
+    return True
+
+def download_json(json_url):
+    response = requests.get(json_url, timeout=5, verify=False)
+    response.raise_for_status()
```

Comment:
We should check that the URL `is not None` before attempting to download. If it `is not None` and we get an error, we should log the error and **not** crash the server; we should just continue using an empty mapping.